receiver/kafkareceiver/header_extraction.go

```diff
@@ -0,0 +1,77 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package kafkareceiver // import "github.com/open-telemetry/opentelemetry-collector-contrib/receiver/kafkareceiver"
+
+import (
+	"fmt"
+
+	"github.com/Shopify/sarama"
+	"go.opentelemetry.io/collector/pdata/plog"
+	"go.opentelemetry.io/collector/pdata/pmetric"
+	"go.opentelemetry.io/collector/pdata/ptrace"
+	"go.uber.org/zap"
+)
+
+var getAttribute = func(key string) string {
+	return fmt.Sprintf("kafka.header.%s", key)
+}
```

Comment:
Is there a reason not to use a normal function? ^^
```suggestion
func getAttribute(key string) string {
	return fmt.Sprintf("kafka.header.%s", key)
}
```